seqr/management/commands/deactivate_project_search.py

```diff
@@ -12,6 +11,7 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument('project')
 
+    @es_only
```

Comment:
the behavior of this manage command for the clickhouse backend can now fully be achieved through the UI so there is no need to maintain it as a separate command